src/utils/davUtils.js

```diff
@@ -23,20 +23,22 @@
 import { generateRemoteUrl } from '@nextcloud/router'
 import { getCurrentUser } from '@nextcloud/auth'
 
-const getRootPath = function() {
+export const getRootPath = function() {
 	if (getCurrentUser()) {
-		return generateRemoteUrl(`dav/files/${getCurrentUser().uid}`)
+		return generateRemoteUrl(`dav${getUserRoot()}`)
 	} else {
 		return generateRemoteUrl('webdav').replace('/remote.php', '/public.php')
 	}
 }
 
-const isPublic = function() {
+export const getUserRoot = function() {
+	return '/files/' + getCurrentUser()?.uid
```

Comment:
throw exception in case there's no current user ? otherwise it would return "/files/" and code might be using it without knowing